### ISSUE TITLE
Cannot enable addon

### ISSUE THREAD

**Issue by Juli2307** [Philippus229/io_scene_osi#1]
Hi, i just installed the addon and when i click on the enable button, it doesn't let me to do it.
It says "Upgrade to 2.8x requiered" but i am already using 2.83.
Am i missing something or do i need to download an older version of blender?

**Comment by Philippus229** [comment]
Yeah, I'm pretty sure I wrote it for Blender 2.79. Just use that and it
should work (technically at least, although I don't remember how functional
it was).

Juli2307 <notifications@github.com> schrieb am So., 15. Nov. 2020, 18:18:

**Comment by Juli2307** [comment]
Ok, i'm gonna try.
I'll tell if it works or not later

**Comment by Juli2307** [comment]
![BlenderAddonError](https://user-images.githubusercontent.com/74505187/99195279-0bfa0380-2785-11eb-8763-a0cda620b3eb.png)

It shows this error.
What does it mean?

**Comment by Philippus229** [comment]
I honestly don't even know why that line is there. It works fine when
commenting it out, but the rotations of some objects are wrong. That might
be because I assumed the the rotation values to be in degrees instead of
radians. Also, it seems to have problems importing some stages. I'm gonna
try to fix these issues real quick and update the files in the repository.
Shouldn't take more than 30 minutes.

Juli2307 <notifications@github.com> schrieb am So., 15. Nov. 2020, 20:57:

**Comment by Philippus229** [comment]
It should work now. Some models still can't be imported, but at least it doesn't crash anymore.

**Comment by Philippus229** [comment]
Nvm, now it just doesn't import most of the objects. Gonna fix it in a few minutes.

**Comment by Philippus229** [comment]
Nvm again, I had just selected the wrong models folder because I was using two versions of Odyssey Editor at the same time. It works now.

**Comment by Philippus229** [comment]
But the code is still terrible and very inefficient. I might rewrite this entire thing and remove the Odyssey Editor stuff entirely, which was initially just to save time because I hadn't planned on releasing this addon at all and then I for some reason didn't change it before uploading it.

**Comment by Juli2307** [comment]
Ok. If you decide to rewrite it tell me please. I'd like to try this addon :)

**Comment by Philippus229** [comment]
Okay, I'll make sure to tell you

**Comment by skavvy** [comment]
i get the same message that Juli got.
